src/tests/suite/daos_test.c

```diff
@@ -333,7 +347,8 @@ main(int argc, char **argv)
 		{"subtests",	required_argument,	NULL,	'u'},
 		{"exclude",	required_argument,	NULL,	'E'},
 		{"filter",	required_argument,	NULL,	'f'},
-		{"dfs",		no_argument,		NULL,	'F'},
+		{"dfs_unit",	no_argument,		NULL,	'I'},
```

Comment:
you should update:
src/tests/ftest/daos_test/daos_core_test.yaml
to add the new test to run in CI